Billing.API/Models/InvoiceListItem.cs

```diff
@@ -13,7 +14,13 @@ public class InvoiceListItem
 
         public InvoiceListItem(string product, string accountId, DateTime date, string currency, double amount, string link)
         {
-            Product = product;
+            Product = product.EqualsIgnoreCase("CD")
+                ? "Doppler"
+                : product.EqualsIgnoreCase("CR")
+                    ? "Relay"
+                    : product.EqualsIgnoreCase("CM")
+                        ? "Client Manager"
+                        : product;
```

Comment:
Nice!

In the future, maybe this could be moved into a dictionary, but by the moment it is ok.

What do you think about changing the format in this way to avoid so much indentation?

```suggestion
            Product = product.EqualsIgnoreCase("CD") ? "Doppler"
                : product.EqualsIgnoreCase("CR") ? "Relay"
                : product.EqualsIgnoreCase("CM") ? "Client Manager"
                : product;
```

Reply:
I thought about using a dictionary, but if the parameter has "doppler" instead of "CD" it will explode.
The alternative would be:

```csharp
var dict = new Dictionary<string, string>(){
   { "CD", "Doppler" },
   { "CR", "Relay" },
   { "CM", "Client Manager" }
}

Product = dict.ContainsKey[product] ? dict[product] : product
```

Reply:
The right way to do it is with _TryGetValue_ as we talked, but by the moment we do not need it, the ternary is fine.